spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/data/TestHelpers.java

```diff
@@ -808,4 +810,11 @@ public static Set<DeleteFile> deleteFiles(Table table) {
 
     return deleteFiles;
   }
+
+  public static Set<String> reachableManifestPaths(Table table) {
+    return StreamSupport.stream(table.snapshots().spliterator(), false)
```

Comment:
There's actually a method in `ManifestFiles` that more efficiently returns just the paths if you want to use it. @amogh-jahagirdar uses it in the non-Spark reachable file implementation: https://github.com/apache/iceberg/pull/5669#discussion_r990676800

Reply:
Not sure if I miss something, but looks like that is for data files of a manifest, while this is for listing manifests?

Reply:
Yeah it's for listing live data file paths for a given manifest.